### DISCUSSION TITLE
TP3 - Archivos a entregar

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#96] [Preguntas y respuestas] [answered]
Una pregunta del tp3. Los ejercicios deben ser todos por separados o todo junto??.
Es decir: hay que subir un archivo que contenga los punto 1.1, 1.2, 1.3, 1.4 y asi con el punto dos o es cada punto un archivo distinto??

**Comment by martinvilu** [comment, marked as answer]
Debieran de quedar, dos archivos, uno para cada punto
